scripts/cybershake/install_cybershake_fault.py

```diff
@@ -167,12 +168,25 @@ def install_fault(
             extended_period=extended_period,
         )
 
+        if not isclose(vm_params_dict["flo"], root_params_dict["flo"]):
+            logger.critical(
+                "The parameter 'flo' doe not match in the VM params and root params files. "
+                "Please ensure you are installing the correct gmsim version"
+            )
+            root_params_dict, fault_params_dict, sim_params_dict, vm_add_params_dict = (
+                None,
+                None,
+                None,
+                None,
+            )
+
         if (
             root_params_dict is None
             and fault_params_dict is None
```

Comment:
or?

Reply:
Or don't we necesarily need all of them?

Reply:
They're only None if there's an nt mismatch in the install_simulation function, or an flo mismatch.
This is just a stop gap till the prepro/install refactor can get done/in